### PR TITLE
Use correct QE curve for R7081

### PR DESCRIPTION
**Pull request by JamesJieranShen**
QE Curve being used is introduced in https://github.com/rat-pac/ratpac-two/pull/43. 
Hey Max  (@masmiley ) -- given you are the original author of this curve, could you confirm if this change is OK?
Ping @llebanowski as well just in case.

### CODE REVIEW

**Comment by masmiley** [comment]
I didn't actually digitize this QE curve, I just copied it in #43 from one ratdb file (`OPTICS.ratdb`) to another (`OPTICS_Photocathode.ratdb`) where it currently resides. I would suggest going to the Hamamatsu datasheets and comparing a digitization to what is currently in ratpac.

**Comment by JamesJieranShen** [comment]
Per @llebanowski's suggestion, updated non-HQE versions to use seperate QE curve, also from the Hamamatsu datasheet. Thanks to @elikeener for preparing the new QE tables!

File: ratdb/OPTICS_Photocathode.ratdb [hunk 1]
```diff
@@ -291,4 +291,59 @@ THICKNESS_value2: [26e-6, 26e-6, ],
...
+                    4.34097381e-03, 3.02350125e-03, 2.32421952e-03, 1.70075600e-03, 1.24442040e-03,
+                    8.95727402e-04, 6.88561683e-04, 4.87538273e-04, 3.62674030e-04, 2.69772513e-04,
+                    2.04002300e-04]
+
+RINDEX_option: "wavelength",
+RINDEX_value1: [60.0, 200.0, 380.0, 395.0, 410.0, 425.0, 440.0, 455.0, 470.0, 485.0, 500.0, 515.0, 530.0, 545.0, 560.0, 575.0, 590.0, 605.0, 620.0, 635.0, 650.0, 665.0, 680.0, 800.0, ],
+RINDEX_value2: [1.0, 1.0, 1.92, 2.18, 2.38, 2.61, 2.70, 2.87, 3.00, 3.00, 3.00, 3.09, 3.26, 3.20, 3.12, 3.06, 3.01, 2.98, 2.96, 2.95, 2.95, 2.95, 2.96, 3.0, ],
+
+
+//RINDEX_option: "wavelength",
```
Comment (tannerbk) [inline ratdb/OPTICS_Photocathode.ratdb:335]: Can you remove the commented out code (I suppose for all cases, in this file).

Comment (JamesJieranShen) [reply ratdb/OPTICS_Photocathode.ratdb]: Fixed in latest commit

**Pull request merged**